site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5397,7 +5397,13 @@ public function alreadyInAQueue($user_id = null) {
         return 0 < count($this->course_db->rows());
     }
 
+    public function lastTimeInQueue($user_id, $queue_code) {
```

Comment:
Functions here should start with a verb, to make it clearer as to what the function is doing, without having to look at it. Does this return the last time the user was in the queue? Does it set something? etc.

For things that just get info from the database, should usually just be `getFoo()`. Change this for the couple of new functions you added below.